tests/test_e2e_60_of_multi_table.py

```diff
@@ -0,0 +1,183 @@
+        response = requests.post(api_url, json=evc)
+        assert response.status_code == 201, response.text
+        data = response.json()
+        assert 'circuit_id' in data
+        time.sleep(10)
+
+        # Add pipeline
+        api_url = KYTOS_API + '/of_multi_table/v1/pipeline'
+        response = requests.post(api_url, json=pipeline)
+        data = response.json()
+        assert response.status_code == 201
+        assert 'id' in data
+
+        # Enabled pipeline
+        api_url = KYTOS_API + '/of_multi_table/v1/pipeline/' + data['id'] + '/enable'
+        response = requests.post(api_url)
+        data = response.json()
+        assert response.status_code == 200
+        time.sleep(10)
+
+        s1 = self.net.net.get('s1')
+        flows_s1 = s1.dpctl('dump-flows').splitlines()
+        for flow in flows_s1:
+            if "table=0" in flow:
+                assert 'actions=resubmit(,1)' in flow
+            if "table=1" in flow:
+                assert 'actions=CONTROLLER:65535' in flow
+            if "table=2" in flow:
+                assert 'dl_type=0x88cc' in flow
+            if "table=3" in flow:
+                assert 'dl_vlan=100 actions=output:"s1-eth2"' in flow
+            if "table=4" in flow:
+                assert 'actions=mod_vlan_vid:100,output:"s1-eth1"' in flow
+
+
+    def test_010_delete_miss_flow(self):
+        """Delete a miss flow so is recreated"""
+        pipeline = {
+            "multi_table": [
+                {
+                    "table_id": 0,
+                    "description": "First table for miss flow entry",
+                    "table_miss_flow": {
+                        "priority": 0,
+                        "instructions": [{
+                            "instruction_type": "goto_table",
+                            "table_id": 1
+                        }],
+                        "match": {"dl_vlan": 101}
+                    },
+                },
+                {
+                    "table_id": 1,
+                    "description": "Second table for coloring",
+                    "napps_table_groups": {
+                        "coloring": ["base"],
+                        "of_lldp": ["base"],
+                        "mef_eline": ["evpl", "epl"],
+                    },
+                },
+            ]
+        }
+
+        # Add pipeline
+        api_url = KYTOS_API + '/of_multi_table/v1/pipeline'
+        response = requests.post(api_url, json=pipeline)
+        data = response.json()
+        assert response.status_code == 201
+        assert 'id' in data
+
+        # Enabled pipeline
+        api_url = KYTOS_API + '/of_multi_table/v1/pipeline/' + data['id'] + '/enable'
+        response = requests.post(api_url)
+        data = response.json()
+        assert response.status_code == 200
+        time.sleep(10)
+
+        s1 = self.net.net.get('s1')
+        flows_s1 = s1.dpctl('dump-flows')
+        assert 'actions=resubmit(,1)' in flows_s1
```

Comment:
I encounter this action instead of the usual `goto_table`, Is `resubmit(,1)` expected in the e2e tests?